test_python/layers/test_matmul_train.py

```diff
@@ -0,0 +1,36 @@
+import tensor_comprehensions as tc
+
+import torch
+import torch.cuda
+from torch.autograd import Variable
+from torch.nn.parameter import Parameter
+import unittest
+
+
+class TestTrainMatMul(unittest.TestCase):
+
+    def test_train_matmul(self):
+        LANG = """
+        def matmul(float(M,N) A, float(N,K) B) -> (output) {
+          output(i, j) +=! A(i, kk) * B(kk, j)
+        }
+        def matmul_grad(float(M,N) A, float(N,K) B, float(M,K) O_grad) -> (A_grad, B_grad){
+          A_grad(i, j) +=! O_grad(i, kk) * B(j, kk)
+          B_grad(i, j) +=! O_grad(kk, j) * A(kk, i)
+        }
+        """
+
+        # NOTE: TC doesn't support padding yet
```

Comment:
This note isnt relevant, can you remove this? :)

Reply:
Yes, I forgot removing.